public/app/features/explore/Explore.tsx

```diff
@@ -182,10 +183,46 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
     }
   };
 
+  /**
+   * Used by Logs details.
+   */
+  analyzeQuery = async (check: string, key: string, value: string) => {
+    let isActive = false;
+    const { queries } = this.props;
+    for (const query of queries) {
+      const ds = await getDataSourceSrv().get(query.datasource);
+      if (hasQueryManipulationSupport(ds, 'analyzeQuery') && ds.analyzeQuery) {
+        isActive = ds.analyzeQuery(query, {
+          check,
+          attributes: {
+            key,
+            value,
+          },
+        });
+      } else {
+        return false;
+      }
+    }
+    return isActive;
+  };
+
+  /**
+   * Used by Logs details.
+   */
+  isFilterLabelActive = async (key: string, value: string) => {
+    return await this.analyzeQuery('HAS_FILTER', key, value);
+  };
```

Comment:
For now we don't. Since the original filter works with `=` and `!=` (or `filter` and `-filter` for Elastic), we check for the presence of either of those.

Reply:
For this use case, we only care about equality filters, which is is what ![imagen](https://github.com/grafana/grafana/assets/1069378/ddf04375-3c8e-4b71-8f00-96326d35c5d8) adds.

When ![imagen](https://github.com/grafana/grafana/assets/1069378/4b3b7de1-7eb3-49ba-b64e-19cd060bfe15) is pressed, the `key => value` combination will not be present in any log line, so it's not necessary to show it as active.